components/form/FCreditCardInput.vue

```diff
@@ -34,6 +34,7 @@ FField.FCreditCardInput(
         FCreditCardIcon(
           v-if="!loading"
           :card-type="creditCard?.type"
+          :isValid="isValidCreditCard(value)"
```

Comment:
ça va partir d'ici, mais pour info ⚠️ au casing des props => kebab-case `:is-valid`